docs/beacon.md

```diff
@@ -0,0 +1,11 @@
+# Beacon v2
+
+1. (First time only:) Run `git submodule init`
+1. Run `git submodule update --remote` to checkout the beaconized data
+1. Start beacon: `docker compose up`
+1. (First time only: )Run: `config/beacon_db_load.sh` to load all data to the db
+1. Beacon should be running at port 5050, try it `curl localhost:5050/api | jq .`
```

Comment:
What should I look for in the response to know that everything is working?

Reply:
The `response.name` should be "EJPRD Swedish Test Beacon".
The info shown here can be configured from `config/beacon_conf.py`, please update as you see fit.

Reply:
That seems to work but the datasets key is an empty array. Is there a way to make sure that the datasets have been properly indexed? Is there a repository with the Docker image?

Reply:
I'm not sure why the `datasets` is empty, but it seems to be the same for other beacons, even if they have public data. 

However, there is probably a bug in our current db set up, will push a fix soon.